packages/sdk/client/src/client/client.ts

```diff
@@ -205,27 +205,26 @@ export class Client {
   /**
    * @deprecated Temporary.
    */
-  get experimental() {
-    // eslint-disable-next-line @typescript-eslint/no-this-alias
-    const self = this;
-    return {
-      get graph() {
-        return self._echoClient.graph;
-      },
-    };
+  get graph() {
+    return this._echoClient.graph;
   }
 
-  // TODO(dmaretskyi): Expose `graph` directly?
-  addSchema(...schemaList: Parameters<RuntimeSchemaRegistry['addSchema']>) {
+  /**
+   * Add schema types to the client.
+   */
+  // TODO(burdon): Check if already registered (and remove downstream checks).
+  addTypes(types: S.Schema<any>[]) {
```

Comment:
array vs spread? spread feels like a cleaner api

Reply:
i don't like spread (varargs!) in many places it is any array already; i think its less ambiguous; supports adding options if needed later.